changes/tasks.py

```diff
@@ -1458,9 +1358,9 @@ def handle_hsm_error(self, user_id, source_id, identity_uuid):
         language = identity["details"]["lang_code"].lower().replace("_", "-")
         with translation.override(language):
             text = translation.ugettext(
-                "Sorry but we can't send WhatsApp msgs to this phone. We'll "
-                "send your MomConnect msgs on SMS. To stop dial "
-                "%(optout_ussd)s, for more dial %(popi_ussd)s (Free)."
+                "Sorry we can't send WhatsApp msgs to this phone. We'll send your "
+                "MomConnect msgs on SMS. To stop dial *134*550*1#, for more dial "
+                "*134*550*7#"
```

Comment:
We probably don't want to change this content, or if we do, we would want to do it in a separate PR, not as part of the cleanup. This is what's creating all the changes in the PO files.

Reply:
ahh I see, okay will reverse this